.travis.yml

```diff
@@ -205,3 +218,9 @@ script:
       bjam arm-tests warnings-as-errors=on ssl=$ssl crypto=$crypto variant=test_arm $toolset target-os=linux link=static testing.launcher="sudo cp -R bin rootfs/; sudo chroot rootfs";
     fi'
   - cd ..
+
+# test build for iOS, using x86_64 to avoid the need to build boost from source for arm
+  - 'if [ "$iphone" == "1" ]; then
+      brew upgrade boost;
+      bjam -j3 variant=test_debug warnings-as-errors=on asserts=on toolset=darwin-x86_64~iphonesim link=static;
```

Comment:
you know, I didn't tried that, I was only testing that iOS build are not broken, but it the tests run, it would be nice, I till test.